WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -387,6 +387,14 @@ class WPMainActivityViewModel @Inject constructor(
         selectedSiteRepository.removeSite()
     }
 
+    fun onMainBottomSheetCreated() {
+        _mainActions.value?.let { actions ->
+            if (actions.any { it is AnswerBloggingPromptAction }) {
+                analyticsTracker.track(Stat.BLOGGING_PROMPTS_CREATE_SHEET_CARD_VIEWED)
```

Comment:
While testing this, the event was not tracked the **first time I tap the FAB**. 

I believe it is because the `onFabClicked` function calls `loadMainActions` which will asynchronously load `_mainActions`, so this `onMainBottomSheetCreated` function might run before that is updated.

https://github.com/wordpress-mobile/WordPress-Android/blob/58ae19e3e69321c902aa6220311d41a217159e09/WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt#L264-L270

Reply:
I added a suggestion in the new place so I will close this conversation.